packages/node/src/sdk/index.ts

```diff
@@ -140,13 +140,19 @@ function _init(
   const scope = getCurrentScope();
   scope.update(options.initialScope);
 
+  if (options.spotlight && options.integrations.some(({ name }) => name !== SPOTLIGHT_INTEGRATION_NAME)) {
```

Comment:
m: My brain is a bit fried already today but the the `.some` check isn't correct right? We only want to add the integration if it hasn't been added to `options.integrations` before. I think we can use `.find` instead

Reply:
You are right, I got confused! Fix coming out that said I prefer `.some()` to `.find()` as we don't really need the first found value, we just want to check if there is _anything_ that matches the criteria. They probably work the same underneath but they don't have as you can optimize `.some()` in different ways if you wanted to.